src/components/RoomNameInput.js

```diff
@@ -108,7 +96,7 @@ class RoomNameInput extends Component {
                 prefixCharacter={CONST.POLICY.ROOM_PREFIX}
                 placeholder={this.props.translate('newRoomPage.social')}
                 onChange={this.setModifiedRoomName}
-                value={this.state.roomName.substring(1)}
+                defaultValue={this.props.initialValue.substring(1)} // Since room name always starts with a prefix, we omit the first character
```

Comment:
@eVoloshchak NAB: This makes it easier to read the comment
```suggestion

                // Since room name always starts with a prefix, we omit the first character to avoid displaying it twice.
                defaultValue={this.props.initialValue.substring(1)}
```

Reply:
> @eVoloshchak NAB: This makes it easier to read the comment

This results in a following linter error: `Expected no line gap between “onChange” and “defaultValue”  react/jsx-props-no-multi-space`.
But if you remove the gap, it breaks another rule: `Expected line before comment  lines-around-comment`.
I couldn't find a way to satisfy both `lines-around-comment` and `jsx-props-no-multi-space` (they seem to contradict each other), please let me know if I'm missing something

Reply:
Ohhh interesting. I'll raise it on slack, but for now please just update this comment.
Everything else looks great!

```suggestion
                defaultValue={this.props.initialValue.substring(1)} // Since the room name always starts with a prefix, we omit the first character to avoid displaying it twice.
```